exporter/exporterhelper/internal/sizer/logs_sizer.go

```diff
@@ -37,7 +37,7 @@ type LogsBytesSizer struct {
 // This is derived from opentelemetry-collector/pdata/internal/data/protogen/logs/v1/logs.pb.go
 // which is generated with gogo/protobuf.
 func (s *LogsBytesSizer) DeltaSize(newItemSize int) int {
-	return 1 + newItemSize + math_bits.Len64(uint64(newItemSize|1)+6)/7 //nolint:gosec // disable G115
```

Comment:
To see the error, try remove the conversion to `uint64`:
What is here: `math_bits.Len64(x|1+6)/7`
What should be: `(math_bits.Len64(x|1) + 6) / 7`